docs/adr/0006-rapid-releases-and-moving-towards-a-monorepo.md

```diff
@@ -0,0 +1,121 @@
+# Rapid releases and moving towards a monorepo
+
+* Status: proposed
+* Deciders:
+  * Sync team: ✅ bendk, ✅ tarikeshaq, lougeniaC64, ✅ ddurst, mhammond
+  * Release management: ✅ rvandermeulen
+  * Release Engineering: cyrus-bond
+  * Firefox Android:  ✅ csadilek
+  * Firefox iOS: ✅ nish
+  * Firefox Desktop: cyrus-bond, sylvestre
+
+* Date: Dec 19, 2022
+* Feedback deadline: Jan 16, 2022
+
+## Context and Problem Statement
+
+Application services currently has a long cycle time for code changes.  We make a release from main on an ad-hoc schedule, averaging about 3-4 releases a month.  Our consumer applications take our releases also on an ad-hoc basis, `firefox-android` takes our releases fairly frequently, `firefox-ios` less frequently, and desktop very infrequently.  This means that code changes take a relatively long time to be released to the public.  For example when the `v96.0.1` release was taken by `firefox-android`, the new commits for that release ranged from 3-18 days old.
```

Comment:
A bit of a noob question, but I have no idea how app-services works :)..

This paragraph talks about different products taking app-services releases at different frequencies. However most of the rest of the doc seems to be centered around the problem of needing to co-ordinate PRs to all land at the same time (presumably so they're all using the same version?). Those seem contradictory to me, do all products need to be using the same version at once?